src/commands/distribute/release.ts

```diff
@@ -69,7 +69,7 @@ export default class ReleaseBinaryCommand extends AppCommand {
   @hasArg
   public storeName: string;
 
-  @help("Release notes text")
+  @help("Release notes text (markdown supported, 5000 characters max)")
```

Comment:
Markdown is not supported for this option, actually. See https://github.com/microsoft/appcenter-cli/issues/1184#issuecomment-789300556

Reply:
Add his note about 5000 characters to release-notes-file, and for that option you can specify that markdown is supported.

Reply:
```suggestion
  @help("Release notes text")
```